tree/master/cloud/src/deployments/Static Content/Matter Center Assets/Matter Landing Assets/scripts/MatterLanding.js

```diff
@@ -542,20 +567,33 @@ function displayContent() {
 		    	
 		    // Cache the pin element
 	    	oElement = $("#PinMatter");
-			$(".matterDescriptionTitle, .matterProfileTitle").click(function () {
+			$(".matterDescriptionTitle, .matterProfileTitle, .matterExtraPropertyTitle").click(function () {
```

Comment:
Logic to display correct tab on selection of tab like Matter Properties, Matter Description etc.